### PR TITLE
Rep profile oj

### PR DESCRIPTION
**Pull request by ojones311**

### CODE REVIEW

File: pages/profile/[id].js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if(memberInfo){
+            return <BasicInfo data={memberInfo}/>
+        }else{
+            return <div></div>
+        }
+    }
+    if(!id) return <h1> Loading...</h1>
+
+    return(
+        <div>
```
Comment (vonbarown) [inline pages/profile/[id].js]: can you add the navbar

File: pages/profile/[id].js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        votingPos.then(({results}) => setVotingPositions(results[0].votes));
+        memberInformation.then(({results}) => setMemberInfo(results[0]))
+    }, [])
+
+    const renderBasicInfo = () => {
+        if(memberInfo){
+            return <BasicInfo data={memberInfo}/>
+        }else{
+            return <div></div>
+        }
```
Comment (vonbarown) [inline pages/profile/[id].js]: You can return the empty the null state directly 
```suggestion
        if(memberInfo){
          return <BasicInfo data={memberInfo}/>
        }
       return <div></div>
```

File: components/Profile/VotingPosition.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const VotingPosition = ({data}) => {
+    if(data.amendment.length > 0){
```
Comment (vonbarown) [inline components/Profile/VotingPosition.js]: you should also check for ``` data.bill.bill_id``` there are voting positions showing up with any id or information

File: components/Profile/VotingPosition.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        return (
+            <>
+            <div className='bill-info'>
+                <p>{data.date}</p>
+                <p>Bill Id:{' '}{data.bill.bill_id}</p>
+                <h2>Description:{' '}{data.description}</h2>
+                <h3>Title: {' '}{data.bill.title}</h3>
+                <p>How Did They Vote: {' '}{data.position}</p>
+                <p> Vote Result: {' '}{data.result}</p>
+            </div>
```
Comment (vonbarown) [inline components/Profile/VotingPosition.js]: same here, to remove the empty bills
```suggestion
          {data.bill.bill_id && (
          <div className="bill-info">
            <p>{data.date}</p>
            <p>Bill Id: {data.bill.bill_id}</p>
            <h2>Description: {data.description}</h2>
            <h3>Title: {data.bill.title}</h3>
            <p>How Did They Vote: {data.position}</p>
            <p> Vote Result: {data.result}</p>
          </div>
        )}
```

File: components/Profile/BasicInfo.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+const BasicInfo = ({data}) => {
+
+    useEffect(() => {
+        console.log('data',data)
+    })
+   
+    return(
+        <div>
+            <h2>{data.first_name}{" "}{data.last_name}</h2>
+            <p>D.O.B:{" "}{data.date_of_birth}</p>
```
Comment (jenesh) [inline components/Profile/BasicInfo.js]: Write out `Date of Birth`

**Comment by vonbarown** [comment]
run the command ```npm run lint``` in your terminal there are some linter errors

File: components/Profile/BasicInfo.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+            <p>Party:{" "}{data.current_party}</p>
```
Comment (jenesh) [inline components/Profile/BasicInfo.js]: Spell out the part instead of R/D

File: components/Profile/BasicInfo.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+            <p>Website:{" "}<a href={data.url}>{data.url}</a></p>
```
Comment (jenesh) [inline components/Profile/BasicInfo.js]: Open the link in a new tab and on hover make it turn blue w/ underline like a normal a tag.

File: components/Profile/BasicInfo.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+            <a href={data.rss_url}>{data.rss_url}</a>
```
Comment (jenesh) [inline components/Profile/BasicInfo.js]: The RSS feed should be styled somehow.

File: components/Profile/VotingPosition.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+        )
+    }else{
+        return (
+            <>
+            <div className='bill-info'>
+                <p>{data.date}</p>
+                <p>Bill Id:{' '}{data.bill.bill_id}</p>
+                <h2>Description:{' '}{data.description}</h2>
+                <h3>Title: {' '}{data.bill.title}</h3>
+                <p>How Did They Vote: {' '}{data.position}</p>
```
Comment (jenesh) [inline components/Profile/VotingPosition.js]: This text doesn't make sense change it to something else.

**Review by jenesh — CHANGES_REQUESTED** [review]

**Review by jenesh — APPROVED** [review]

**Pull request merged**